frontend/src/core/comments/components/AddComment.css

```diff
@@ -1,32 +1,53 @@
-.comments-list .add-comment form.container {
+.comments-list .add-comment div.container {
     flex-flow: row nowrap;
     align-items: flex-end;
     padding-bottom: 6px;
 }
 
-.comments-list .add-comment textarea {
+.comments-list .add-comment .editor-wrapper {
     align-items: center;
     background-color: #333941;
     border: none;
     border-radius: 4px;
     color: #FFFFFF;
+    cursor: text;
     display: flex;
     flex-grow: 2;
     font-size: 11px;
-    height: auto;
+    min-height: 26px;
+    max-height: 144px;
     line-height: 24px;
```

Comment:
Let's bump the `line-height` to `26px`, otherwise the text jumps a little when the comment breaks into the 2nd line.